### PR TITLE
Fix #928: Innovatrics Configuration

### LINKED ISSUE #928: Innovatrics Configuration
Add Innovatrics properties and spring configuration.

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: docs/onboarding/Configuration-Properties.md [hunk 1]
```diff
@@ -134,6 +134,23 @@ The Onboarding Server uses the following public configuration properties:
 | `enrollment-server-onboarding.presence-check.iproov.zenid.restClientConfig.proxyUsername`         |                           | Proxy username to be used when calling iProov REST service.                   |
 | `enrollment-server-onboarding.presence-check.iproov.zenid.restClientConfig.proxyPassword`         |                           | Proxy password to be used when calling iProov REST service.                   |
 
+
+## Innovatrics Configuration
+
+| Property                                                                                | Default                   | Note                                                                           |
+|-----------------------------------------------------------------------------------------|---------------------------|--------------------------------------------------------------------------------|
+| `enrollment-server-onboarding.innovatrics.serviceBaseUrl`                               |                           | Base REST service URL for Innovatrics.                                         |
```
Comment (banterCZ) [inline docs/onboarding/Configuration-Properties.md:142]: 👀 Both Innovatrics providers' configuration is united, no discriminator for `presence-check` or `document-verifier`, just the `innovatrics` key.

File: enrollment-server-onboarding-provider-innovatrics/src/main/java/com/wultra/app/onboardingserver/provider/innovatrics/InnovatricsConfig.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpHeaders;
+
+/**
+ * Innovatrics configuration.
+ *
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+@ConditionalOnExpression("""
```
Comment (banterCZ) [inline enrollment-server-onboarding-provider-innovatrics/src/main/java/com/wultra/app/onboardingserver/provider/innovatrics/InnovatricsConfig.java:36]: We do not support mixing one Innovatrics provider with any other implementation.

- [ ] Add to javadoc

**Review by jnpsk — APPROVED** [review]
Nice. 
I was just thinking about using `enrollment-server-onboarding.provider.innovatrics.serviceBaseUrl` instead to be consistent with the package name. Just an idea, nothing I'd insist on.

**Comment by banterCZ** [comment]
> Nice. I was just thinking about using `enrollment-server-onboarding.provider.innovatrics.serviceBaseUrl` instead to be consistent with the package name. Just an idea, nothing I'd insist on.

👍 `provider` sounds good, I will improve it.

**Pull request merged**